### PR TITLE
Add sort by date button and functionality

### PR DESCRIPTION
**Pull request by andrew-ware**
Add sort by date button and functionality

### CODE REVIEW

File: app/src/components/Landing/Stories/middleware/sort.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function sort() {
+  const currIndex = this.options.indexOf(this.storiesSort);
+  let newVal;
+  if (currIndex > -1) {
+    newVal = this.options[(this.options.indexOf(this.storiesSort) + 1) % 2];
```
Comment (jsleathe12295) [inline app/src/components/Landing/Stories/middleware/sort.js]: I think there is an easier way to do this, maybe a boolean instead of an array?

Comment (andrew-ware) [inline app/src/components/Landing/Stories/middleware/sort.js]: You're right. A boolean would be way cleaner.

File: app/src/components/Landing/Stories/middleware/sort.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+function sort() {
```
Comment (jsleathe12295) [inline app/src/components/Landing/Stories/middleware/sort.js]: Looks good!

File: app/src/components/Landing/Stories/middleware/sort.js (hunk 1 above)
Comment (jsleathe12295) [inline app/src/components/Landing/Stories/middleware/sort.js]: Awesome looks good!

**Comment by andrew-ware** [comment]
Great nice changes LGTM

**Pull request merged**